### ISSUE TITLE
Failed to install dependency / Kodi Krypton (Windows Store version)

### ISSUE THREAD

**Issue by gurabli** [dexterke/plugin.video.digi.hu-online#2]
Tried to install from zip file, but failed with the above message. I will try later on CoreElec.

**Comment by dexterke** [comment]
I've tested the plugin on the following platforms:
Kodi 17 on Ubuntu 16.04
Kodi 17 on MacOS X 10.11
Kodi 17 on OSMC (Rasbian Linux for RaspberryPI)
Kodi 17 on Android TV (Android 8)

I do not have a Windows computer, however this is a dependency issue, not really something that I could control. 
What exactly is the error code, or what is failing, "Failed to install dependency" could mean anything.

Try latest version (0.1.0.d), I just made 2 minor changes, commented out line 21 in default.py and lin 9 in addon.xml.

The following addons are listed now as dependencies
xbmc.python
script.common.plugin.cache
script.module.requests
